File: Taller de Haskell/Python/problem5.py
# Comprobador de elegibilidad de impuestos

edad = int(input("Ingrese su edad: "))
ingresos = int(input("Ingrese sus ingresos mensuales (COP): "))

# Condicionales
# Si los ingresos mensuales son superiores a 6000000 debera declarar renta

if edad >= 18 and ingresos >= 6000000:
    print("Debe pagar impuestos")
else:
    print("No está obligado a pagar impuestos")

File: Taller de Haskell/Haskell/problem5.hs
pagaImpuestos :: Int -> Double -> Bool
pagaImpuestos edad ingresos = edad >= 18 && ingresos >= 6000000

main :: IO ()
main = do
    putStrLn "Ingrese su edad: "
    edadStr <- getLine
    putStrLn "Ingrese sus ingresos mensuales: "
    ingresosStr <- getLine
    let edad = read edadStr :: Int
    let ingresos = read ingresosStr :: Double
    if pagaImpuestos edad ingresos
        then putStrLn "Debe pagar impuestos."
        else putStrLn "No esta obligado a pagar impuestos."